conf/dokuwiki.php

```diff
@@ -17,6 +19,7 @@
 $conf['start']       = 'start';           //name of start page
 $conf['lang']        = 'en';              //your language
 $conf['template']    = 'dokuwiki';         //see lib/tpl directory
+$conf['frontEndTemplate']    = '';         //frontend template to use for public pages
```

Comment:
wouldn't it make more sense to specify the backend template and keep the frontend default?